examples/jenkins-master/jenkins-slave-template.json

```diff
@@ -44,7 +50,7 @@
         "name": "${IMAGE_STREAM_NAME}"
       },
       "spec": {
-        "dockerImageRepository": "openshift/${IMAGE_STREAM_NAME}"
+        "dockerImageRepository": "${NAMESPACE}/${IMAGE_STREAM_NAME}"
```

Comment:
Yeah I originally thought that it should be left as is and then talked
myself into it, thinking the change would at least generate the discussion,
as it was a bit atypical from my previous experience.

On Monday, March 14, 2016, Ben Parees <notifications@github.com> wrote:

> In examples/jenkins-master/jenkins-slave-template.json
> <https://github.com/openshift/origin/pull/8009#discussion_r56107955>:
>
> > @@ -44,7 +50,7 @@
> >          "name": "${IMAGE_STREAM_NAME}"
> >        },
> >        "spec": {
> > -        "dockerImageRepository": "openshift/${IMAGE_STREAM_NAME}"
> > +        "dockerImageRepository": "${NAMESPACE}/${IMAGE_STREAM_NAME}"
>
> this is a docker pullspec, not an imagestream, so specifying the namespace
> is the wrong thing to do.
>
> This template is a bit complicated in what it needs to do, it's working
> with both docker images as well as imagestreams, and sharing a single
> parameter for both. We can talk about how to address it tomorrow.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/8009/files#r56107955
>